clang_analyze.sh

```diff
@@ -21,5 +63,6 @@ if [[ -z $nthreads ]]; then
     echo >&2 "Warning: Could not determine number of CPU cores, using $nthreads"
 fi
 
-CC=clang CXX=clang++ scan-build --use-cc=clang --use-c++=clang++ cmake -S "$sourcedir" -B "$builddir" -DCMAKE_BUILD_TYPE=Debug "$@"
-scan-build --use-cc=clang --use-c++=clang++ make -j$(nproc) -C"$builddir"
+CC=clang CXX=clang++ scan-build --use-cc=clang --use-c++=clang++ ${checkers} cmake -S "$sourcedir" -B "$builddir" -DCMAKE_BUILD_TYPE=Debug "$@"
+scan-build --use-cc=clang --use-c++=clang++ ${checkers} make -j$(nproc) -C"$builddir"
```

Comment:
This bug already existed but `$(nproc)` should be `$nthreads` which is computed earlier.

Reply:
Nice catch!